examples/azure-native-sample-project/README.md

```diff
@@ -0,0 +1,42 @@
+# Azure Native sample project
+
+This project creates a VM instance in Azure Cloud using the Pulumi Kotlin Azure native SDK.
+
+## How to run
+
+**See the 
+[Azure Native: Installation & Configuration](https://www.pulumi.com/registry/packages/azure-native/installation-configuration/)
+page in Pulumi Registry for the official documentation.**
+
+1. Install Pulumi (see: [docs](https://www.pulumi.com/docs/install/)).
+   ```bash
+   # on MacOS
+   brew install pulumi/tap/pulumi
+   ```
+2. Log in to your Pulumi account (see: [docs](https://www.pulumi.com/docs/cli/commands/pulumi_login/)).
+   ```bash
+   pulumi login
+   ```
```

Comment:
Could you please add a newline after and before all code-blocks? It improves padding in the rendered document.

### Before

<img width="1046" alt="Screenshot 2023-07-17 at 23 08 08" src="https://github.com/VirtuslabRnD/pulumi-kotlin/assets/4415632/0501b57a-7369-48e9-90da-d8f64dc57eeb">

### After

<img width="1043" alt="Screenshot 2023-07-17 at 23 07 32" src="https://github.com/VirtuslabRnD/pulumi-kotlin/assets/4415632/3142f13c-eabf-4631-923c-239e492b4a6a">